tests/FakeItEasy.Analyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -0,0 +1,282 @@
+﻿namespace FakeItEasy.Analyzer.Tests.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Superclass of all Unit Tests for DiagnosticAnalyzers.
+    /// </summary>
+    public abstract partial class DiagnosticVerifier
+    {
+        /// <summary>
+        /// Get the CSharp analyzer being tested - to be implemented in non-abstract class.
+        /// </summary>
+        /// <returns>The diagnostic analyzer being tested.</returns>
+        protected virtual DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+        {
+            return null;
+        }
+
+        /// <summary>
+        /// Get the Visual Basic analyzer being tested (C#) - to be implemented in non-abstract class.

```

Comment:
I suspect that everything in `Helpers` was provided, but what's this comment mean? "**Visual Basic** analyzer being tested **(C#)**"